test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -7456,42 +7456,126 @@ def test_batchnorm_training(self):
         class MyModule(torch.nn.Module):
             def __init__(self):
                 super(MyModule, self).__init__()
-                self.bn = torch.nn.BatchNorm2d(3, affine=True)
+                self.bn1 = torch.nn.BatchNorm2d(3, affine=False)
+                self.cv1 = torch.nn.Conv2d(3, 3, 10)
+                self.bn2 = torch.nn.BatchNorm2d(3, affine=True)
+                self.cv2 = torch.nn.Conv2d(3, 3, 10)
+                self.bn3 = torch.nn.BatchNorm2d(3, affine=False)
 
             def forward(self, x):
-                bn = self.bn(x)
-                return bn
+                x = self.bn1(x)
+                x = self.cv1(x)
+                x = self.bn2(x)
+                x = self.cv2(x)
+                x = self.bn3(x)
+                return x, self.bn3.running_mean, self.bn3.running_var
 
-        model = MyModule()
-        x = torch.randn(10, 3, 128, 128)
+        x = torch.randn(10, 3, 64, 64) * 2
+        model_export = MyModule()
+        model_export.train()
+        outs = model_export(x)
+        pytorch_out = [out.detach().numpy() for out in outs]
+        ort_sess = convert_to_onnx(model_export, input=(x,), opset_version=self.opset_version, example_outputs=outs,
+                                   training=torch.onnx.TrainingMode.TRAINING,
```

Comment:
why calling convert_to_onnx and compare results explicitly? Would adding training flag to `run_test` enough to resolve this?